torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -140,8 +150,11 @@ def _retrieve_or_adapt_input_to_graph_set(
     ):
         sequence_elements: List[
             Union[
-                graph_building.TorchScriptTensor,
-                Tuple[graph_building.TorchScriptTensor, ...],
+                onnxscript.function_libs.torch_lib.graph_building.TorchScriptTensor,
```

Comment:
```suggestion
                onnxscript_graph_building.TorchScriptTensor,
```

Looks like this can be done because it was imported. Elsewhere as well

Reply:
Indeed, after the classes split, `import onnxscript` and `onnxscript.function_libs.torch_lib.graph_building` could be imported at the beginning. I can use the shorter version from now on